### PR TITLE
added FocalLoss

### PR DESCRIPTION
**Pull request by SuzannaLin**
Hi Yassine!

I added focal loss to the losses.py file and modified train.py and model.py. I hope I didn't forget to adjust anything.
Since I counted my object pixels separately, I hard coded them in the train.py script. I will work on an object pixel counter within the script, but since my computer is training now, I will do that later and commit when I have the code.

Thanks so much!

Suzanna

### CODE REVIEW

**Comment by yassouali** [comment]
Hi @SuzannaLin 

Thanks for the pull request, everything seems correct, I just placed softmax_helper in losses instead of train.
